src/docs/sphinx/quickstart.rst

```diff
@@ -177,40 +177,72 @@ container. Instructions for this process are located :ref:`here <docker-label>`.
 Building Serac
 --------------
 
-Serac uses a CMake build system that wraps its configure step with a script
-called ``config-build.py``.  This script creates a build directory and
-runs the necessary CMake command for you. You just need to point the script
-at the generated or a provided host-config. This can be accomplished with
-one of the following commands:
+Serac is a CMake project. Configure with at least the ``-C``
```

Comment:
One of the large benefits of `config-build.py` is that it takes the CMake executable from the host-config. Unfortunately, CMake has wildly differing behavior based on what CMake executable is used.

This section needs to say to refer to the host-config on the given platform to see what CMake to use.

Reply:
> CMake has wildly differing behavior based on what CMake executable is used.

Would it be beneficial to require a more recent version of CMake, so that there are fewer versions to worry about? 3.18 or later, would also have the benefit of native CUDA language support. I haven't noticed any issues with serac + cmake v3.20 - 3.26.

Reply:
Unfortunately newer CMake's are not always better. It is often the case that their behavior changes and bugs are reintroduced.

The only safe thing I have found is to lock it down to a blessed version of CMake to avoid wasting time at the developer level.

Reply:
I'll include that in the quickstart guide, then. What is the blessed version of CMake?

Reply:
on LC its 3.18.0